src/legacy/core_plugins/console/server/api_server/spec/generated/explain.json

```diff
@@ -22,8 +22,7 @@
       "POST"
     ],
     "patterns": [
-      "{indices}/_explain/{id}",
-      "{indices}/{type}/{id}/_explain"
+      "{indices}/_explain/{id}"
```

Comment:
There are a few more instances like this in `bulk.json`, `create.json`, `delete.json`, `exists_source.json`, `indices.exists_type.json`, etc.

Reply:
I don't think this work should block this PR - but we can instead create an issue for Elasticsearch UI team.